examples/kitchen-sink/packages/ui/tsup.config.ts

```diff
@@ -0,0 +1,13 @@
+import { defineConfig } from "tsup";
+
+export default defineConfig({
```

Comment:
Also, now that we have a config here you can remove the args from `package.json` `build` and `dev` scripts and just call 'tsup and `tsup --watch` respectively. You will also need to update this config to support overrides from the cli args to make that watch option work!

```js
import { defineConfig, Options } from "tsup";

export default defineConfig((options: Options) => ({
  [defaults]
  ...options,
}));
```